app/uk/gov/hmrc/rdsdatacacheproxy/Module.scala

```diff
@@ -40,5 +40,6 @@ class Module extends AppModule:
       bind[AuthAction].to(classOf[DefaultAuthAction]),
       bind[DirectDebitController].toSelf,
       bind[RdsDataSource].to(datasource),
-      bind[CisMonthlyReturnSource].to(cisDatasource)
+      bind[CisMonthlyReturnSource].to(cisDatasource),
```

Comment:
update ModuleSpec to test bind of CisMonthlyReturnSource to either CisRdsStub or CisDatacacheRepository